app/Libraries/Markdown/HtmlBlockRenderer.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the GNU Affero General Public License v3.0.
+// See the LICENCE file in the repository root for full licence text.
+
+namespace App\Libraries\Markdown;
+
+use League\CommonMark\Block\Element\AbstractBlock;
+use League\CommonMark\Block\Renderer\BlockRendererInterface;
+use League\CommonMark\Block\Renderer\HtmlBlockRenderer as BaseHtmlBlockRenderer;
+use League\CommonMark\ElementRendererInterface;
+use League\CommonMark\Util\ConfigurationAwareInterface;
+use League\CommonMark\Util\ConfigurationInterface;
+
+class HtmlBlockRenderer implements BlockRendererInterface, ConfigurationAwareInterface
```

Comment:
a new block type is probably better?

Something like <code>```</code> for code block but different character.

Reply:
Definitely agree with something that isn't inline html.